content/en/digital_certificate.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Digital Certificate
+status: Feedback Appreciated
+category: Concept
+---
+
+## What it is
+A digital certificate or  a public key certificate, which is issued by the Certificate Authority, is an electronic password that allows an entity to exchange data or resources over the internet in a secured manner. This is achieved by proving the authenticity of the user through the use of encryption methods and set of roles, policies, resources used to manage, use and revoke this digital certificate and encryptions. Much as a passport certifies one’s identity, the purpose of a digital certificate is to establish the identity of users within the ecosystem.
+
+## Problem it addresses
+Digital certificates help solve web-based communication security issues. Networks developed without much thought to security. TCP/IP, the most prevalent communication protocol, was designed around precepts of robust communication rather than secure transactions. While the OSI network model designed a layer between the application and network that could manage security handshakes, TCP/IP left everything above the communication layer to the application. Consequently each application developed its own negotiation strategy. Rather than building a generic, reusable security mechanism, applications relied on proprietary or embedded techniques for shuffling account information across the network to the platform or application for validation.
+
+## How it helps
+Digital certificates are the most practical option to securing your corporate data online. They encrypt your internal and external communications to avoid outsiders and phishers from stealing sensitive information. From small to large businesses, public key certificates can still provide you with the same quality of encryption. You can easily centralize the management of your certificates with the help of platforms such as managed public key infrastructure. Digital certificates are so scalable, you can even use it to secure your personal devices. Institutions known as public trusted Certificate Authorities handle the issuance of digital certificates making it reliable.  By ultimately encrypting your company website and digitally signing your documents and emails, you are projecting a positive reputation to your clients.
```

Comment:
```suggestion
Digital certificates are the most practical option for securing data online. Certificates encrypt internal and external communications to prevent outsiders and phishers from stealing sensitive information. They provide the same encryption quality for small to large businesses alike. 

Digital certificates are highly scalable and can even be used to secure personal devices. Institutions known as public trusted certificate authorities handle the issuance of digital certificates, ensuring they are reliable. Certificate management can be easily centralized through managed public key infrastructure platforms. 
```

Reply:
Mostly edits here again (content didn't change). I did remove "By ultimately encrypting your company website and digitally signing your documents and emails, you are projecting a positive reputation to your clients," as this seems to be providing advice or judging those who don't do that. I think we need to keep it highly factual. 

Also, do we need "They provide the same encryption quality for small to large businesses alike.?" Why would the quality change depending on your size? After all, it's the same technology. I think we should skip that sentence. What do you think?